docker/start.sh

```diff
@@ -22,6 +22,7 @@ else
     php artisan migrate
     # call the email template seeder to updateOrCreate without truncating first
     DISABLE_TRUNCATE=true php artisan db:seed --class=EmailTemplatesSeeder
+    php artisan validation:generate-logs
```

Comment:
uses updateOrCreate so should be safe - otherwise have to manually apply some data seeding